tools/recetox_msfinder/recetox_msfinder.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="recetox_msfinder" name="RECETOX MsFinder" version="@TOOL_VERSION@+galaxy0">
+    <description>Peak Annotation</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator" />
+
+    <expand macro="requirements" />
+    <command>
+        <![CDATA[
+        sh ${write_param} &&
+        mono /MsFinder/MsFinder/bin/Debug/MsfinderConsoleApp.exe annotate -i "$input_data" -m "MSFINDER.INI" -o "$annotated_data"        
```

Comment:
please use single-quotes around all path-like or text-like parameters

Reply:
resolved in [feb89bf](https://github.com/RECETOX/galaxytools/pull/313/commits/feb89bf7c3effb19096caadbb7de383cc3254627)